ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -73,7 +74,13 @@ internal fun Markdown(
     val root = parser.parse(text) as Document
 
     val density = LocalDensity.current
-    val paragraphPadding = with(density) { style.lineHeight.toDp() }
+    val paragraphPadding = with(density) {
+        if (style.lineHeight.type == TextUnitType.Sp) {
```

Comment:
This was causing some snapshots not to work because the unit wasn't Sp. For now, I'm just making the spacing 0 if the unit is not Sp, which didn't seem to happen in production, only on previews.